src/Sylius/Bundle/ApiBundle/Swagger/ProductImageDocumentationNormalizer.php

```diff
@@ -45,17 +38,18 @@ public function normalize($object, $format = null, array $context = [])
         $enums = $this->filterProvider->provideShopFilters();
         $enums = array_keys($enums);
 
-        $params = $docs['paths'][self::SHOP_ITEM_PATH]['get']['parameters'];
+        $shopProductImagePath = $this->apiRoute . '/shop/product-images/{id}';
```

Comment:
I would leave suffix as a const

Reply:
I also think of it but it was a private const so it wasn't editable nor accessible.